mobile/lib/components/ra_player/ra_player_handler.dart

```diff
@@ -108,18 +181,16 @@ class AudioPlayerHandler extends BaseAudioHandler with SeekHandler {
     return PlaybackState(
       controls: [
         // MediaControl.rewind,
-        // if (_player.playing) MediaControl.pause else MediaControl.play,
+        if (mediaKind.value == MediaKind.recording)
+          if (_player.playing) MediaControl.pause else MediaControl.play,
         // MediaControl.stop,
         // MediaControl.fastForward,
         if (_player.playing) MediaControl.stop else MediaControl.play,
       ],
```

Comment:
This looks quite messy. Also, how about adding rewind and fast forward to recording controls? (sorry for not using code suggestion tool, GitHub didn't let me use it)

`      controls: mediaKind.value == MediaKind.recording`
`          ? [`
`              MediaControl.rewind,`
`              if (_player.playing) MediaControl.pause else MediaControl.play,`
`              MediaControl.fastForward,`
`            ]`
`          : [`
`              if (_player.playing) MediaControl.stop else MediaControl.play,`
`            ],`

Reply:
Great suggestion, and I'll do it in (I hope) even more readable way!